perf/Microsoft.Azure.WebJobs.Perf/packages.config

```diff
@@ -1,13 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Data.Edm" version="5.8.1" targetFramework="net45" />
-  <package id="Microsoft.Data.OData" version="5.8.1" targetFramework="net45" />
-  <package id="Microsoft.Data.Services.Client" version="5.8.1" targetFramework="net45" />
+  <package id="Microsoft.Azure.KeyVault.Core" version="2.0.4" targetFramework="net45" />
+  <package id="Microsoft.Data.Edm" version="5.8.2" targetFramework="net45" />
+  <package id="Microsoft.Data.OData" version="5.8.2" targetFramework="net45" />
+  <package id="Microsoft.Data.Services.Client" version="5.8.2" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.2.3" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
-  <package id="System.Spatial" version="5.8.1" targetFramework="net45" />
-  <package id="WindowsAzure.Storage" version="7.2.1" targetFramework="net45" />
+  <package id="System.ComponentModel.EventBasedAsync" version="4.3.0" targetFramework="net45" />
+  <package id="System.Dynamic.Runtime" version="4.3.0" targetFramework="net45" />
+  <package id="System.Linq.Queryable" version="4.3.0" targetFramework="net45" />
+  <package id="System.Net.Requests" version="4.3.0" targetFramework="net45" />
```

Comment:
Microsoft.Data.Services.Client increased its dependencies in this version -- that's where these new ones came from.